package.json

```diff
@@ -24,6 +24,7 @@
     "start": "pm2-runtime start pm2.config.js",
     "heroku-postbuild": "echo 'Skipping build'",
     "build": "node scripts/prod.js",
+    "pg:build": "pgtyped -c ./packages/server/postgres/pgtypedConfig.js",
```

Comment:
what do you recommend as a pattern for calling this? should we call it on ever server restart?
perhaps we could use pm2 to run a dedicated instance that watches a directory & runs it whenever that dir changes?
we can leave that piece out of scope for this PR, but it'll be good to think about for a future PR

Reply:
pgtyped cli has watch mode, and i was thinking we could call this from `dev.js`. disclaimer: i haven't tried invoking watch mode manually yet, so i can't yet vouch for its reliability.

https://pgtyped.now.sh/docs/cli